### ISSUE TITLE
The repository link from metacpan doesn't work

### ISSUE THREAD

**Issue by benkasminbullock** [rcoscali/JSON.minify#3]
You have JSON_minify in the Makefile.PL:

https://metacpan.org/source/RCOSCALI/JSON_minify-1.1/Makefile.PL#L13

That should be JSON.minify I think.

Also please note that the rt.cpan.org bug tracker is due to close down so I put the bug report here.

**Comment by rcoscali** [comment]
Thanks for your report!
This fix will have as a side effect (but is it a side effect or is it really what you want to achieve ?) to rename module from JSON_minify to minify and to migrate it in the JSON namespace. 
I'm not opposed to this idea, that seems good at first. However I need to check with JSON namespace authors if it will not ne problematic for them..?
Or is there some other problem that JSON_minify trigger (I didn't notived)?

However thanks for your report, I'll try your proposal and will post results of my experiments.

Regards

Rémi

**Comment by rcoscali** [comment]
Or perhaps I'm happily scrambling perl and java (and even perhaps some others xD).
I check and get back to you ...
;-)

**Comment by benkasminbullock** [comment]
Yes it will have that side effect if you insist that `$PACKAGE` must not be altered or you cannot use another variable for the `NAME` field in Makefile. Why you insist on that is not clear to me. The github repository doesn't have to have the same name as the CPAN distribution.

Anyone can put anything under the JSON:: namespace on CPAN, it doesn't belong to one person.

**Comment by rcoscali** [comment]
This is because I didn't used perl for ... 6 years ... outch
Then I'm not sure about the problem I could create, and I don't want to annoy other module developpers.
Or perhaps I fear for nothing ... and I can't create some annoying side effects ...???
(Actually I wrote this module 6 years ago, and forgot to push it into cpan)

**Comment by benkasminbullock** [comment]
In this case the repo field is pointing to a nonexistent place.

Go to the left side here:

https://metacpan.org/pod/JSON_minify

The reason is that it has _ not . in the name. The github repo is `JSON.minify`.

Incidentally you have some problems with testers:

http://matrix.cpantesters.org/?dist=JSON_minify+1.1

I haven't looked into what happened yet.

**Comment by rcoscali** [comment]
OK Got it !!

I fix it immediatly. Just give me a few hours to check what is done on cpantesters.org side ...

Thanks !
